src/pages/_app.tsx

```diff
@@ -1,10 +1,22 @@
-import '@/styles/globals.css'
+import Head from 'next/head';
+import '@/styles/tailwind-globals.css';
+import '@/styles/globals.css';
 import "@/styles/global-background-color.scss"
 import "@/styles/global-font-style.scss"
 import "@/styles/global-text-color.scss"
 
 import type { AppProps } from 'next/app'
 
 export default function App({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+  return(
+  <>
+    <Head>
+      <title>Taotify</title>
+      <meta name="description" content="Generated by create next app" />
```

Comment:
Since you have added this description, please make it meaningful.

Reply:
update description with "Discover, Play, and Enjoy Your Favorite Music with Taotify - Your Ultimate Music Companion"